R/retro.R

```diff
@@ -72,6 +74,13 @@ SS_doRetro <-
 #'   endyrvec = endyrvec,
 #'   legendlabels = paste("Data", 0:-5, "years")
 #' )
+#' 
+#' ## run retrospectives in parallel
+#' future::plan(future::multisession)
```

Comment:
Just a thought:
In my workflow for something else that I am using furrr for, I have:
```
ncores <- parallel::detectCores() - 1 
future::plan(multisession, workers = ncores)
```
I saw it on some tutorial for working with furrr awhile back, but I'm not sure if it's a best practice. The workflow I have is using a 64-core azure VM, but I guess if others are using something as large, it might be worth adding a similar line to the example

Reply:
@e-gugliotti-NOAA, I'm trying to understand the default number or workers associated with `future::plan(future::multisession)`. It looks like maybe it's all the available cores, so providing an example where the user has control as you suggest makes good sense.
I see here: https://parallelly.futureverse.org/reference/availableCores.html#avoid-ending-up-with-zero-cores-1 a suggestion to use `ncores <- parallelly::availableCores(omit = 1)`.

I was a little concerned about having r4ss depend on a bunch of additional packages, but it looks like {furrr} brings in {globals}, {parallelly}, and {future} which are all part of the well-supported futureverse collection and not that large.

Reply:
I am definitely not a {furrr} expert either! Just enthusiastic about getting this stuff running faster, and potentially moving some of these really slow things like jittering to the cloud. I will add this line to the examples for both functions.